components/templates/src/builtins/rss.xml

```diff
@@ -6,25 +6,35 @@
         {%- elif section.title %} - {{ section.title }}
         {%- endif -%}
       </title>
-        <link>{%- if section -%}
-            {{ section.permalink | escape_xml | safe }}
-          {%- else -%}
-            {{ config.base_url | escape_xml | safe }}
-          {%- endif -%}
-        </link>
-        <description>{{ config.description }}</description>
-        <generator>Zola</generator>
-        <language>{{ lang }}</language>
-        <atom:link href="{{ feed_url | safe }}" rel="self" type="application/rss+xml"/>
-        <lastBuildDate>{{ last_updated | date(format="%a, %d %b %Y %H:%M:%S %z") }}</lastBuildDate>
-        {%- for page in pages %}
-        <item>
-            <title>{{ page.title }}</title>
-            <pubDate>{{ page.date | date(format="%a, %d %b %Y %H:%M:%S %z") }}</pubDate>
-            <link>{{ page.permalink | escape_xml | safe }}</link>
-            <guid>{{ page.permalink | escape_xml | safe }}</guid>
-            <description>{% if page.summary %}{{ page.summary }}{% else %}{{ page.content }}{% endif %}</description>
-        </item>
-        {%- endfor %}
+      <link>
+        {%- if section -%}
+          {{ section.permalink | escape_xml | safe }}
+        {%- else -%}
+          {{ config.base_url | escape_xml | safe }}
+        {%- endif -%}
+      </link>
+      <description>{{ config.description }}</description>
+      <generator>Zola</generator>
+      <language>{{ lang }}</language>
+      <atom:link href="{{ feed_url | safe }}" rel="self" type="application/rss+xml"/>
+      <lastBuildDate>{{ last_updated | date(format="%a, %d %b %Y %H:%M:%S %z") }}</lastBuildDate>
+      {%- for page in pages %}
+      <item>
+          <title>{{ page.title }}</title>
+          <pubDate>{{ page.date | date(format="%a, %d %b %Y %H:%M:%S %z") }}</pubDate>
+          <author>
+            {%- if page.authors[0] -%}
```

Comment:
same here

Reply:
I think you're right -- fixed in new push.